.github/workflows/test.yaml

```diff
@@ -38,10 +39,69 @@ jobs:
             reports/jest/junit.xml
             reports/junit/*.xml
           comment_mode: off
-      - name: Send Slack notification with job status.
-        if: always()
+  save-cache:
+    name: Save Cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Cache Docker images.
+        uses: ./
+        with:
+          key: docker-cache-test
+      - name: Build an empty test Docker image.
+        run: |
+          docker build --tag empty - <<<'FROM scratch
+            LABEL description="empty image"'
+  restore-cache:
+    name: Restore Cache
+    needs:
+      - save-cache
+    runs-on: ubuntu-20.04
+    permissions: {}
+    steps:
+      - name: Check out repository.
+        uses: actions/checkout@v3.0.2
+      - name: Load test cache.
```

Comment:
nit: Consider "Load Docker images" for parity with the corresponding `save-cache` step.